contentgrid-hateoas-spring/src/main/java/com/contentgrid/hateoas/spring/pagination/SlicedResourcesAssembler.java

```diff
@@ -77,16 +71,16 @@ private <R> SlicedModel<R> addPaginationLinks(SlicedModel<R> model, PaginationCo
         }
 
         if (controls.hasPrevious()) {
-            model.add(createLink(base, controls.previous(), IanaLinkRelations.PREV));
+            model.add(createLink(base, controls.previous().orElse(null), IanaLinkRelations.PREV));
```

Comment:
Could also be updated to make it a bit shorter, and without the wrapping `if`

```java
controls.previous().ifPresent(previousPagination -> model.add(createLink(base, previousPagination, IanaLinkRelations.PREV))
```

Reply:
done